packages/forma-36-codemod/bin/cli.js

```diff
@@ -28,10 +28,10 @@ function checkGitStatus(force) {
     }
   }
 
-  if (!clean) {
+  if (!clean && !force) {
     console.log(
       chalk.yellow(
-        'Before we continue, please stash or commit your git changes.',
+        'Before we continue, please stash or commit your git changes or use the --force flag to override this check',
```

Comment:
```suggestion
        'Before we continue, please stash or commit your changes.\nIf you want to override this check, use the --force flag',
```

Wdyt about doing this? I'm suggesting it so we avoid repeating "or" that much

This should render in two lines like:

```
Before we continue, please stash or commit your changes.
If you want to override this check, use the --force flag
```